ebmlite/decoding.py

```diff
@@ -15,9 +15,9 @@
 from datetime import datetime, timedelta
 import struct
 
-#===============================================================================
+# ===============================================================================
```

Comment:
Not sure the space is necessary. In any case, the lines should be shortened.

Reply:
Eh, it's technically supposed to have a space after the hash.  It doesn't need it, but my IDE has been complaining.